incubator/group/types.go

```diff
@@ -0,0 +1,86 @@
+package group
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/params"
+	"github.com/cosmos/cosmos-sdk/x/params/subspace"
+	"github.com/cosmos/modules/incubator/orm"
+	"gopkg.in/yaml.v2"
+
+	"time"
+)
+
+type GroupID uint64
+
+func (g GroupID) Byte() []byte {
```

Comment:
This should be plural `Bytes`, but I believe it's an ORM thing and we can address in a separate PR

Reply:
ja, plural. will add to #24